torch/csrc/autograd/python_engine.cpp

```diff
@@ -92,12 +92,13 @@ PyObject *THPEngine_run_backward(THPEngine *self, PyObject *args, PyObject *kwar
   unsigned char keep_graph = 0;
   unsigned char create_graph = 0;
   PyObject *inputs = nullptr;
+  unsigned char allow_unreachable = 1;
```

Comment:
The C default is different than the one in Python. Can we make this consistent?

Reply:
I set it that way because .backward defaults that to True. But changing this also makes sense. :)

Reply:
This doesn't matter for `.backward`, right? Is it even an argument?

Reply:
`.backward` also calls `Variable._execution_engine.run_backward`, so it is an arg for the C apt.